packages/mobile/src/backup/BackupSocialIntro.tsx

```diff
@@ -3,30 +3,41 @@ import colors from '@celo/react-components/styles/colors'
 import { fontStyles } from '@celo/react-components/styles/fonts'
 import * as React from 'react'
 import { WithNamespaces, withNamespaces } from 'react-i18next'
-import { Image, ScrollView, StyleSheet, Text } from 'react-native'
+import { ActivityIndicator, Image, ScrollView, StyleSheet, Text } from 'react-native'
 import SafeAreaView from 'react-native-safe-area-view'
 import { NavigationInjectedProps } from 'react-navigation'
 import { connect } from 'react-redux'
 import CeloAnalytics from 'src/analytics/CeloAnalytics'
 import { CustomEventNames } from 'src/analytics/constants'
 import componentWithAnalytics from 'src/analytics/wrapper'
-import { exitBackupFlow } from 'src/app/actions'
+import { exitBackupFlow, navigatePinProtected } from 'src/app/actions'
 import { Namespaces } from 'src/i18n'
 import backupIcon from 'src/images/backup-icon.png'
 import { headerWithBackButton } from 'src/navigator/Headers'
-import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { navigateHome } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { RootState } from 'src/redux/reducers'
 
+interface StateProps {
+  doingPinVerification: boolean
```

Comment:
I don't think this screen needs this pin check. It's redundant because the user needs to enter their pin to reach this far.

Reply:
Actually not, I've disabled the check if they are going only to introduction. This, to my opinion, makes more sense - so the user can read about the feature before entering the PIN. What do you think? Also I've covered this case in the video.